lib/index.ts

```diff
@@ -798,6 +891,19 @@ export class Worker {
 					objectWithLinks as any,
 					patch,
 				);
+
+				if (
+					typeContract.slug === 'link' &&
+					insertSession.actor.slug !== 'user-admin'
+				) {
```

Comment:
Can drop this? Already have the same check in the `validateOrgMembership()` function.

Reply:
👍